src/js/components/Header.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+class Header extends React.Component {
```

Comment:
I wonder which Header this should be?

Could we be more specific on the naming?

Maybe `SiteHeader` ? Not sure if this is better though.